charts/trino/values.yaml

```diff
@@ -348,6 +348,9 @@ service:
   annotations: {}
   type: ClusterIP
   port: 8080
+# -- The port the service listens on.
```

Comment:
If the property is commented out, it won't be added to the README. Can we have it without a default value (empty string)? Also, please document the default behavior - that the node port is selected by random, you can even reference the official Kubernetes documentation, using Markdown links: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport-custom-port

Reply:
Or start the comment with a fully qualified property name, like `# service.nodePort -- The port the service listens on.`. I'm not sure if that'll work when the property is commented out, but let's see.

Reply:
I standardized the comment by indicating `# service.nodePort` and also included the official documentation.

I also tested by inserting a string [""] and the behavior is null, that is, a port is randomly assigned.